landing-page/types/alces-branding/content/styles/branding.css

```diff
@@ -32,31 +40,43 @@ body {
     font-weight: bold;
     font-size: larger;
 }
+.navbar .dropdown .btn-group .nav-menu-button {
+    padding-right: 0;
+}
+.navbar .dropdown .btn-group .dropdown-toggle-split {
+    padding-right: 24px;
+}
 
 .navbar.brandbar .navbar-nav-container,
 .navbar.productbar .navbar-nav-container {
     justify-content: end;
 }
+.navbar .navbar-nav .nav-item {
+    border-right-width: 1px;
+    border-right-style: dotted;
+    border-right-color: rgb(88, 103, 117);
+}
 .navbar .navbar-nav .nav-link {
     color: white !important;
     font-size: 12px;
     text-transform: uppercase;
     font-weight: normal;
-    border-right-width: 1px;
-    border-right-style: dotted;
-    border-right-color: rgb(88, 103, 117);
     padding: 8px 24px 8px 24px;
 }
+.navbar .navbar-nav .dropdown-item.nav-link {
+    padding-left: 8px;
+    padding-right: 8px;
+}
```

Comment:
I think the default padding-{left,right} of 24px is probably better.  Can you remove this please?